style.css

```diff
@@ -0,0 +1,124 @@
+:root {
+  --d-green: #253d2c;
+  --l-green: #68ba7f;
+  --dark: #000000;
+  --white: #ffffff;
+  --grey: #d9d9d9;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+i {
+  padding: 0.2rem;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: var(--white);
+}
+
+.Header-top {
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--grey);
+}
+
+.Icon1 {
+  margin: 0 4rem;
+  padding: 0.5rem;
+}
+
+.Icon1 a {
+  font-size: 0.8rem;
+  color: var(--dark);
+}
+
+.Social-links {
+  margin: 0 4rem;
+  padding: 0.5rem;
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+}
+
+.logo img{
```

Comment:
you can use rem for this width and height if you can't because the image sizes are not that big where it will make affect